src/style.css

```diff
@@ -1 +1,39 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
According to the checklist, avoid using tag names for styling. It's better to use classes to style elements to ensure that the styles are not too broadly applied and can be easily maintained.

Reply:
there is no tag name styling